### ISSUE TITLE
Integrate with OSX tags

### ISSUE THREAD

**Issue by srustamo** [hut/ranger#444]
Currently implemented tagging doesn't make use of OSX system-wide file tagging. Is it possible to use OSX tags instead or in addition to `ranger` tags? CLI tag handling in OSX seems possible, see [this](https://github.com/jdberry/tag) for sample implementation.

**Comment by hut** [comment]
Yes it's possible in theory, but I practically, no major ranger dev uses OSX so nobody would implement this

**Comment by charignon** [comment]
I can look into it: @hut, how would you suggest displaying them?
